### ISSUE TITLE
Searchbar Unhighlighted fails AA/AAA from WCAG

### ISSUE THREAD

**Issue by codyduong** [codyduong/codyduong.dev#13]
![image](https://user-images.githubusercontent.com/24193268/210007801-59b9f84b-90f2-429e-ad91-a378035ec5e6.png)
